packages/grid_client/tests/modules/zmount.test.ts

```diff
@@ -1,58 +1,39 @@
-import { Zmount } from "../../src";
-
-let zmount: Zmount;
-
-beforeEach(() => {
-  zmount = new Zmount();
-});
-describe("Zmount module", () => {
-  test("Zmount instance is of type Zmount.", () => {
-    expect(zmount).toBeInstanceOf(Zmount);
-  });
+import { validate } from "class-validator";
 
-  test("Min value for size.", () => {
-    const size = 100 * 1025 ** 2;
-
-    zmount.size = size;
-
-    const result = () => zmount.challenge();
-
-    expect(result).toThrow();
-  });
-
-  test("Max value for size.", () => {
-    const size = 100 * 1025 ** 4;
-
-    zmount.size = size;
+import { Zmount } from "../../src";
 
-    const result = () => zmount.challenge();
+describe("Zmount Class", () => {
+  let zmount: Zmount;
 
-    expect(result).toThrow();
+  beforeEach(() => {
+    zmount = new Zmount();
   });
 
-  test("Size doesn't accept decimal value.", () => {
-    const size = 1.5;
-
-    zmount.size = size;
+  describe("size property validation", () => {
+    it("should fail validation if size is less than the minimum", async () => {
+      // Less than 100 MB
+      expect(() => (zmount.size = 50 * 1024 ** 2)).toThrow();
+    });
 
-    const result = () => zmount.challenge();
+    it("should fail validation if size is greater than the maximum", async () => {
+      // Greater than 10 TB
+      expect(() => (zmount.size = 15 * 1024 ** 4)).toThrow();
+    });
 
-    expect(result).toThrow();
-  });
-
-  test("Size empty value.", () => {
-    const result = () => zmount.challenge();
+    it("should pass validation if size is within the valid range", async () => {
+      zmount.size = 5 * 1024 ** 3;
 
-    expect(result).toThrow();
+      const errors = await validate(zmount);
+      expect(errors.length).toBe(0);
+    });
   });
 
-  test("Size negative value.", () => {
-    const negative_size = -1;
-
-    zmount.size = negative_size;
-
-    const result = () => zmount.challenge();
+  describe("challenge method", () => {
+    it("should return the size as a string", () => {
+      zmount.size = 5 * 1024 ** 3;
 
-    expect(result).toThrow();
+      const result = zmount.challenge();
+      expect(result).toBe("5368709120");
```

Comment:
without checking if size is int , the challenge can return any special chars in the string, no?